docs/flink_job_cluster_guide.md

```diff
@@ -0,0 +1,108 @@
+# Create Flink job clusters with Helm Chart
+
+## Overview
+
+This Helm Chart is an addition to the [existing](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/tree/master/config/samples) way of deploying Flink job clusters.
+
+## Prerequisites
+
+
+- Fink Operator Image Version:  ```gcr.io/flink-operator/flink-operator:v1beta1-6``` follow these [instructions](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/tree/master/helm-chart/flink-operator) to deploy the Operator
+- Flink Image Version: ```flink:1.9.3``` or ```flink:latest```
+- Helm Version 2.x or 3.x
+
+
+## Installing the Chart
+
+The instructions to install the Flink Job Cluster chart:
+
+1. Clone the repository to your local machine, which has access to your running kubernetes cluster.
+  ```bash
+  git clone https://github.com/GoogleCloudPlatform/flink-on-k8s-operator.git
+  ```
+2. Navigate to the following folder: ```/flink-on-k8s-operator/helm-chart```
+
+3. Use the following command to install the Flink job cluster chart:
+  ```bash
+  helm template --namespace example-namespace flink-job-cluster ./flink-job-cluster -f ./flink-job-cluster/values.yaml |kubectl apply -f -
+  ```
+
+***Note*** the ```values.file``` in ```/flink-on-k8s-operator/helm-chart/flink-job-cluster/``` is just an example configuration. You can use your own values.yaml if you wish and edit the parts that you want to change. The current values.yaml has the minimum configuration requirements enabled for the Flink job cluster to start successfully. 
+
+## Uninstalling the Chart
+
+To uninstall your release:
+
+1. Navigate to the following folder: ```/flink-on-k8s-operator/helm-chart```
+
+2. Use the following command to uninstall the Flink job cluster chart:
+  ```bash
+  helm template --namespace example-namespace flink-job-cluster ./flink-job-cluster -f ./flink-job-cluster/values.yaml |kubectl delete -f -
+  ```
+
+## Check the Flink job cluster
+
+After using the helm template command, the following resources will be deployed
```

Comment:
After running the command, I noticed only a JM is running, no TM and job.

```
$ kpods

NAMESPACE               NAME                                                        READY   STATUS    RESTARTS   AGE
default                 flink-job-cluster-jobmanager-7ccb67f784-ldr4s               1/1     Running   0          2m
flink-operator-system   flink-operator-controller-manager-85cffbc4fb-wvkwl          2/2     Running   0          20m
```

In the operator pod log, I noticed this error:

```
2020-08-03T16:50:15.777Z        ERROR   controllers.FlinkCluster        Failed to reconcile     {"cluster": "default/flink-job-cluster", "error": "Deployment.apps \"flink-job-cluster-taskmanager\" is invalid: spec.template.spec.containers[0].volumeMounts[0].name: Not found: \"config\""}
```

Reply:
I am going to remove the configmap mount since it was needed for our specific purposes. I'll leave in the possibility to mount files. I believe it is up to the users that use the chart whether they want to mount something or not.